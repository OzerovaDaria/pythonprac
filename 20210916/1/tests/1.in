21+88*45/7, 'aakda'+'nckcll', 2.52362
