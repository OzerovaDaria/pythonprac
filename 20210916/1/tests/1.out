586.7142857142857 aakdanckcll 2.52362
